### PR TITLE
document make test-acceptance-api,cli,webui

### PR DESCRIPTION
**Pull request by phil-davis**
Issue #140 

We have introduced these 3 make file targets ``test-acceptance-api`` ``test-acceptance-cli`` and ``test-acceptance-api`` in core. They are also implemented in some apps, and we will gradually implement this "standard" through all apps that have acceptance tests.

### CODE REVIEW

File: modules/developer_manual/pages/core/acceptance-tests.adoc [hunk 1]
```diff
@@ -240,14 +240,23 @@ sudo -u www-data $installation_path/occ maintenance:install \
   --database-pass=` --admin-user='admin' --admin-pass='admin'
 ----
 
+=== Types of Acceptance Tests
+
+
+There are 3 types of acceptance tests; API, CLI and webUI. API tests test the ownCloud public APIs.
```
Comment (settermjd) [inline modules/developer_manual/pages/core/acceptance-tests.adoc]: This is a little nitpicky, perhaps, but listing the description of the three types of tests as a bulleted list should increase readability.

Comment (settermjd) [reply modules/developer_manual/pages/core/acceptance-tests.adoc]: Great to hear @phil-davis. Thanks.

Comment (settermjd) [reply modules/developer_manual/pages/core/acceptance-tests.adoc]: Um, did I miss something, as the text hasn't changed.

Comment (phil-davis) [reply modules/developer_manual/pages/core/acceptance-tests.adoc]: Sad, I guess I pushed too quickly and did not look at the message. The changes will be sitting in the office. I will have to push them in the next days when I go by.

File: modules/developer_manual/pages/core/acceptance-tests.adoc [hunk 2]
```diff
@@ -235,81 +234,85 @@ mysql -u root -h localhost -e "drop user oc_admin@localhost"
...
 
+=== Running Acceptance Tests for a Suite
 
 Run a command like the following:
 
-.. code-block:: bash
+[source,bash]
+----
+sudo -u www-data make test-acceptance-api BEHAT_SUITE=apiTags
+sudo -u www-data make test-acceptancecli BEHAT_SUITE=cliProvisioning
```
Comment (phil-davis) [inline modules/developer_manual/pages/core/acceptance-tests.adoc]: I have pushed a commit to fix this
s/test-acceptancecli/test-acceptance-cli

GitHub is currently having problems, so sometimes I see the commit and sometimes not.

Comment (phil-davis) [inline modules/developer_manual/pages/core/acceptance-tests.adoc]: I have pushed a commit to fix this
s/test-acceptancecli/test-acceptance-cli

GitHub is currently having problems, so sometimes I see the commit and sometimes not.

Comment (phil-davis) [inline modules/developer_manual/pages/core/acceptance-tests.adoc]: I have pushed a commit to fix this
s/test-acceptancecli/test-acceptance-cli

GitHub is currently having problems, so sometimes I see the commit and sometimes not.

Comment (phil-davis) [inline modules/developer_manual/pages/core/acceptance-tests.adoc]: I have pushed a commit to fix this
s/test-acceptancecli/test-acceptance-cli

GitHub is currently having problems, so sometimes I see the commit and sometimes not.

Comment (settermjd) [reply modules/developer_manual/pages/core/acceptance-tests.adoc]: Ah, I see.

**Comment by phil-davis** [comment]
Also ported across to the "old" documentation repo PR https://github.com/owncloud/documentation/pull/4468

**Pull request merged**